devicetypes/Orvibo/Orvibo-Motion-Sensor.src/Orvibo-Motion-Sensor.groovy

```diff
@@ -0,0 +1,126 @@
+/**
+ *  Orvibo Motion Sensor
+ *
+ *  Copyright 2016 Chris Kitch
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+    definition (name: "Orvibo Motion Sensor", namespace: "Orvibo", author: "jk0218.zhang@samsung.com", vid: "generic-motion-2", ocfDeviceType: "x.com.st.d.sensor.motion") {
+		capability "Motion Sensor"
+		capability "Battery"
+
+        fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001", outClusters: "", model: "895a2d80097f4ae2b2d40500d5e03dcc", manufacturer: "ORVIBO"
+	}
+    
```

Comment:
You will need to add a basic tile definition in order for the device to function in ST Classic which is necessary for certification